web/src/main/webapp/menu.ts

```diff
@@ -88,7 +88,6 @@ export class TopSubMenu implements IHtmlElement {
         this.outer = document.createElement("table");
         this.outer.classList.add("menu", "hidden");
         this.tableBody = this.outer.createTBody();
-        this.outer.onmouseleave = () => this.hide();
```

Comment:
This line is not needed?

Reply:
No it isn't needed, it's handled in the TopMenu cell now. The cell in the TopMenu contains the submenu (in the DOM). When the mouse leaves that cell (or any of its children), the submenu disappears. This is actually the same behaviour as many other top menu's in applications, as far as I know.